src/aequitas/bias.py

```diff
@@ -184,16 +324,335 @@ def get_disparity_predefined_groups(self, df, ref_groups_dict, key_columns=None,
+
+        # run SciPy statistical significance test between each group and
+        # reference group
+        for attr_val, eq_var in sample_dict.items():
+            _, difference_significance_p = stats.ttest_ind(
+                sample_dict[ref_group], sample_dict[attr_val], axis=None,
+                equal_var=eq_variance_dict[attr_val], nan_policy='omit')
+
+            measure = "".join(measure.split('binary_'))
+
+            # add column to crosstab to indicate statistical significance
+            disparity_df.loc[disparity_df['attribute_value'] == attr_val, measure + '_significance'] = \
+                difference_significance_p
+
+        return disparity_df
+
+
+
+    @classmethod
+    def get_statistical_significance(cls, original_df, disparity_df, ref_dict,
+                                     score_thresholds=None, model_id=1,
+                                     attr_cols=None, aplha=5e-2):
+        '''
+
+        :param original_df: a dataframe containing a required raw 'score' column
+            and possible raw 'label_value' column.
+        :param disparity_df: Expansion of get_crosstabs() output with additional
+            columns for disparity metrics and each metric's reference group to
+            which significance must be added
+        :param ref_dict: Dictionary indicating reference group for each
+            attribute/ measure combination
+        :param score_thresholds: a dictionary { 'rank_abs':[] , 'rank_pct':[], 'score':[] }
+        :param model_id: (Future functionality) ID(s) of models for which to check
+            statistical significance
+        :param attr_cols: Columns indicating attribute values in original_df
+        :param aplha: Level at which to determine statistical significance.
+            Default is 5e-2 (0.05).
+
+        :return: dataframe with same number of rows as the input (crosstab)
+            but with additional disparity metrics columns, ref_group_values, and
+            statisitcal significance (p-values) of specific metrics.
+        '''
+        if 'label_value' not in original_df.columns:
+            raise ValueError(
+                "Column 'label_value' not in dataframe. Label values are "
+                "required for computing statistical significance of supervised "
+                "metrics.")
+
+        if attr_cols is None:
+            non_attr_cols = [
+                'id', 'model_id', 'entity_id', 'score', 'label_value',
+                'rank_abs', 'rank_pct']
+            # index of the columns that are attributes
+            attr_cols = original_df.columns[~original_df.columns.isin(non_attr_cols)]
+
+        for col in attr_cols:
+            # find the priors_df
+            col_group = original_df.fillna({col: 'pd.np.nan'}).groupby(col)
+            counts = col_group.size()
+
+        # check if all attr_cols exist in df
+        check = [col in original_df.columns for col in attr_cols]
+        if False in check:
```

Comment:
I don't think there's any need for `check`:

```suggestion
        if any(col not in original_df.columns for col in attr_cols):
```